setup.sh

```diff
@@ -8,19 +8,15 @@ echo 'Installing macOS packages...'
 brew update
 brew install node
 brew install mongodb
-brew install grunt
```

Comment:
Grunt's [already in our devDependencies](https://github.com/uclaradio/uclaradio/blob/0109458f4c42e022d397981c293af9c4abe8bb55/package.json#L74)? I'll upgrade it to the latest version though